glide-core/src/socket_listener.rs

```diff
@@ -117,9 +117,15 @@ impl UnixStreamListener {
                     return ReadSocketClosed.into();
                 }
                 Ok(_) => {
-                    return match self.rotating_buffer.get_requests() {
-                        Ok(requests) => ReceivedValues(requests),
-                        Err(err) => UnhandledError(err.into()).into(),
+                    match self.rotating_buffer.get_requests() {
+                        Ok(requests) => {
+                            if !requests.is_empty() {
+                                // continue to read from socket
```

Comment:
the comment isn't in the right place, should be above 126

Reply:
correct, fixed